src/test/groovy/cz/smarteon/loxone/app/LoxoneAppTest.groovy

```diff
@@ -7,7 +7,9 @@ import spock.lang.Specification
 class LoxoneAppTest extends Specification implements SerializationSupport {
```

Comment:
FYI the tests are in the middle of migration into kotlin - no need to rework this, just for the future...